jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
That's a bit of a grey area, given that Jaeger UI doesn't support arbitrary references today. My thoughts are:
  * if you have child-of within the same trace, then remove that reference and populate parent_id instead
  * if you have more than one child-of, use the first for parent_id, keep others are references
  * if you don't have child-of, but have follows-from in the same trace, then still populate parent_id, but _keep the reference_ (this is a bit weird)
  * if you have  reference to a _different_ trace ID, do not populate parent_id, keep it as a reference

This sounds fairly convoluted, I am open to suggestions. We can implement just the first point for now and punt on the rest, which is probably what we've done in other clients so far.